### DISCUSSION TITLE
HDMI Driver BuiltIn

### DISCUSSION THREAD

**Discussion by ghost** [CosmosOS/Cosmos#2948] [Ideas]
An HDMI Driver

**Comment by retrievaaaa** [comment]
Do you know what HDMI is? Spoiler: It starts with C and ends with onnector

**Reply by AnErrupTion** [reply]
C#onnector

**Comment by AnErrupTion** [comment]
VGA means 2 things: the connector, and the graphics device. A "HDMI driver" doesn't make sense, as you can't drive a connector. However, you can make a driver for another graphics device than VGA. That includes a basic Cirrus or QXL device, a VMware SVGA II device (which is already built into Cosmos) or even VirtIO if you or anybody else are feeling adventurous.

**Reply by ghost** [reply]
Yes I need a driver that works on a GTX 1660 Super For A Console That Runs From 720P To 1080P As Of I Do Not Know How To Make A GUI

**Comment by zarlo** [comment]
driving the HDMI signal is the GPUs job so you just need a GPU that is supported and HDMI will work

might need to make a driver to use all that HDMI can do like get screen sizes and some DRM things

**Reply by ghost** [reply]
yes that is kind of what i am looking for driver to use all that HDMI can do like get screen sizes and some DRM things

**Comment by valentinbreiz** [comment]
> driving the HDMI signal is the GPUs job so you just need a GPU that is supported and HDMI will work

not totally true, I don't know if it's common but I remember that on the PS4 we had to work on an HDMI driver implementation for Linux in addition to the GPU driver to make it work
